specification/indices/migrate_to_data_stream/IndicesMigrateToDataStreamRequest.ts

```diff
@@ -21,6 +21,7 @@ import { RequestBase } from '@_types/Base'
 import { IndexName } from '@_types/common'
 
 /**
+ * Convert an index alias to a data stream.
  * Converts an index alias to a data stream.
```

Comment:
> * Converts an index alias to a data stream.

This is another one where I don't think repeating the summary adds a lot of value to the description.